### PR TITLE
Add my details to metadata

### PR DESCRIPTION
**Pull request by twinkarma**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 version = 0.0.1
-author = Example Author
-author_email = author@example.com
+author = Twin Karmakharm
+author_email = t.karmakharm@sheffield.ac.uk
 description = Perform basic calculations in python
 long_description = file: README.md
 long_description_content_type = text/markdown
-url = https://github.com/<owner-account>/python-calculator
+url = https://github.com/tkarma/python-calculator
```
Comment (ns-rse) [inline setup.cfg]: I think the URL here might be wrong, should it perhaps be....
```suggestion
url = https://github.com/twinkarma/python-calculator
```

Also applies to the below URL.

**Review by ns-rse — CHANGES_REQUESTED** [review]
Thanks for updating the information Twin, I noticed a small problem with the URLs you've used, I've put a suggestion in, I hope they're useful.

File: setup.cfg (hunk 1 above)
Comment (twinkarma) [reply setup.cfg]: Good spot! Thanks @ns-rse!

**Review by ns-rse — APPROVED** [review]
Thanks for correcting that so quickly @twinkarma this looks good to merge now.

**Pull request merged**